sdk/storage/azure-storage-blob/src/test/java/com/azure/storage/blob/AzuriteTest.groovy

```diff
@@ -0,0 +1,226 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.azure.storage.blob
+
+import com.azure.storage.blob.models.BlobStorageException
+import com.azure.storage.blob.specialized.BlobClientBase
+import com.azure.storage.blob.specialized.BlobLeaseClientBuilder
+import com.azure.storage.blob.specialized.SpecializedBlobClientBuilder
+import com.azure.storage.common.StorageSharedKeyCredential
+import spock.lang.Unroll
+
+class AzuriteTest extends APISpec {
+    String azuriteEndpoint = "http://127.0.0.1:10000/devstoreaccount1"
+    StorageSharedKeyCredential azuriteCredential = new StorageSharedKeyCredential("devstoreaccount1", "Eby8vdM02xNOcqFlqUwJPLlmEtlCDXJ1OUzFT50uSRZ6IFsuFq2UVErCz4I6tq/K1SZFPTOtr/KBHBeksoGMGw==")
```

Comment:
If this is a real key, it needs to be in an environment variable and should be rotated. If it's some fake key value, can you please add a comment saying so.

Reply:
It's a fake key, will add documentation that it is.